integ-test/src/test/java/org/opensearch/sql/sql/IdentifierIT.java

```diff
@@ -64,6 +64,40 @@ public void testMultipleQueriesWithSpecialIndexNames() throws IOException {
     queryAndAssertTheDoc("SELECT * FROM test.two");
   }
 
+  @Test
+  public void testDoubleUnderscoreIdentifierTest() throws IOException {
+    new Index("test.twounderscores")
+            .addDoc("{\"__age\": 30}");
+    final JSONObject result = new JSONObject(executeQuery("SELECT __age FROM test.twounderscores", "jdbc"));
+
+    verifySchema(result,
+            schema("__age", null, "long"));
+    verifyDataRows(result, rows(30));
+  }
+
+  @Test
+  public void testMetafieldIdentifierTest() throws IOException {
```

Comment:
The parser will fail.  There's no workaround for prometheus data sources for _id, _score, _index, _maxscore and _sort.  But it will work for _other_ fields that begin with an underscore (e.g. _routing or __fieldname) that is not strictly defined as an opensearch meta-field.  
In other words: this is an improvement on the existing functionality... which would just fail for ALL fields starting with an underscore.